ckanext/canada/public/static/licenses.json

```diff
@@ -163,5 +163,50 @@
     "title_fra": "Licence du gouvernement ouvert – Île-du-Prince-Édouard",
     "url": "https://www.princeedwardisland.ca/en/information/finance/open-government-licence-prince-edward-island",
     "url_fra": "https://www.princeedwardisland.ca/fr/information/finances/licence-du-gouvernement-ouvert-ile-du-prince-edouard"
+  },
+  {
+    "domain_content": true,
+    "domain_data": true,
+    "domain_software": false,
+    "family": "",
+    "id": "nb-oglnb",
+    "is_okd_compliant": false,
+    "is_osi_compliant": false,
+    "maintainer": "",
+    "status": "active",
+    "title": "Open Government Licence - New Brunswick",
+    "title_fra": "Licence du gouvernement ouvert - Nouveau-Brunswick",
+    "url": "http://www.snb.ca/e/2000/data-E.html",
+    "url_fra": "http://www.snb.ca/f/2000/data-F.html"
+  },
+  {
+    "domain_content": true,
+    "domain_data": true,
+    "domain_software": false,
+    "family": "",
+    "id": "yk-oglyk",
+    "is_okd_compliant": false,
+    "is_osi_compliant": false,
+    "maintainer": "",
+    "status": "active",
+    "title": "Open Government Licence – Yukon",
+    "title_fra": "Licence du gouvernement ouvert – Yukon",
+    "url": "https://open.yukon.ca/data/open-government-licence-yukon",
+    "url_fra": "https://open.yukon.ca/data/fr/gouvernement-ouvert-licence-du-yukon"
+  },
+  {
+    "domain_content": true,
+    "domain_data": true,
+    "domain_software": false,
+    "family": "",
+    "id": "qc-oglqc",
```

Comment:
why not use the standard code for this one instead? Have they modified the terms from cc-by?

Reply:
If this one is cc-by and attributes Quebec, will it work if another province wants the CC with another attribution?